x/interchainquery/keeper/abci.go

```
@@ -4,15 +4,14 @@
 	"encoding/hex"
 	"time"
 
+	"cosmossdk.io/math"
```

golangci-lint (1.62.2)

File is not `gci`-ed with --skip-generated -s standard -s default -s blank -s dot -s prefix(cosmossdk.io) -s prefix(github.com/cosmos/cosmos-sdk) -s prefix(github.com/cometbft/cometbft) -s prefix(github.com/cosmos/ibc-go) -s prefix(github.com/quicksilver-zone/quicksilver) --custom-order

(gci)